src/pages/edit/FilterSettingsDialog.js

```diff
@@ -165,18 +166,24 @@ const FilterSettingsDialog = ({
                             >
                                 {i18n.t('Cancel')}
                             </Button>
-                            <Button
-                                onClick={() => {
-                                    if (!filtersSelectable) {
-                                        setSelected([])
-                                    }
-                                    onConfirm(filtersSelectable, selected)
-                                }}
-                                primary
-                                type="button"
+                            <Tooltip
+                                content={i18n.t(
+                                    'Cannot confirm changes while offline'
```

Comment:
Shouldn't this tooltip conditionally rendered based on `!filtersSelectable` or the online status to not show when online?

Reply:
The tooltip is for when the button is disabled due to being offline.

Reply:
I don't think so. I am only adding functionality for offline. Nothing else has changed unless I'm missing something?

Reply:
hm, I thought, since this is wrapping the "confirm" button, which I thought would be visible both online and offline, that the tooltip would then also be (incorrectly) visible when online. The button has a `if (!filtersSelectable)` condition, but the tooltip has none. But maybe Im missing something, hard to judge when you just look at this in isolation.